sos/policies/package_managers/flatpak.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Red Hat, Inc. Jose Castillo <jcastillo@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.policies.package_managers import PackageManager
+
+
+class FlatpakPackageManager(PackageManager):
+    """Package Manager for Flatpak distributions
+    """
+
+    query_command = 'flatpak list --columns=name,version,branch'
+    query_path_command = ''
+    files_command = ''
+    verify_command = ''
+    verify_filter = ''
+
+    def _parse_pkg_list(self, pkg_list):
+        for line in pkg_list.splitlines()[1:]:
+            pkg = line.split("\t")
```

Comment:
Unfortunately I haven't found a way to format the output of the 'flatpak list' command natively, so I had to separate by tab

Reply:
sounds good to me

Reply:
There is one mistake in line 25. Output of flatpak list is without header so you don't need to cut first line, because now you are cutting first line of output, so you are cutting one flatpak package from list

Reply:
@dzalezak I have the header in every system where I run 'flatpak list', i.e.:

```
[root@dulcinea ~]# flatpak list
Name                           Application ID                                 Version           Branch               Origin                        Installation
Slack                          com.slack.Slack                                4.33.84           stable               flathub                       system
Podman Desktop                 io.podman_desktop.PodmanDesktop                1.2.0             main                 podmandesktop-origin          system
Freedesktop Platform           org.freedesktop.Platform                       22.08.14          22.08                flathub                       system
Mesa                           org.freedesktop.Platform.GL.default            23.1.4            22.08                flathub                       system
```

Which version are you running?

Reply:
In terminal there is header, but when you are parsing output from flatpat list there is none. You can try it with `flatpak list > a.txt` .

Reply:
Or `flatpak list | cat`

Reply:
Nice catch! I'll fix the code and push it again